build/common.sh

```diff
@@ -317,6 +317,10 @@ function kube::build::has_docker() {
   which docker &> /dev/null
 }
 
+function kube::build::has_ip() {
+  ip -V | grep -o 'iproute2' &> /dev/null
```

Comment:
nit, `-o` does nothing here. If we're going to do this, we might as well make it clear by using the full flag (`-Version`).

I still feel this adds pointless complexity over `command -v ip` for Linux.
This is a small thing and not that big a deal, but I think it's still worth striving for simplicity and readability.

I won't block on my complaint since it's so minor.